packages/gatsby-cli/src/reporter/redux/internal-actions.ts

```diff
@@ -286,6 +286,26 @@ export const endActivity = ({
     }
   }
 
+  // Show message if they haven't already enabled the feature and the js build
+  // duration is greater than 20 seconds (much longer than normal.
+  if (
+    activity.id === `webpack-develop` &&
+    !process.env.GATSBY_EXPERIMENT_DEVJS_LAZY &&
+    durationMS > 20
```

Comment:
Is it 3s or 20s? I'd be curious to know the _ideal_ candidate, and I think we can err on the side of slightly noisier to begin with:

```suggestion
    durationS > 3
```